### PR TITLE
Custom repo wizard

### PR DESCRIPTION
**Pull request by matoval**
# Description

Complete adding custom repos to both create and update image wizard

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```
Comment (adeliaferreira) [inline .husky/pre-commit:2]: should it be included on this commit?

**Comment by adeliaferreira** [comment]
This PR contains 94 commits, we should keep feature branch focused in feature, can you rebase your changes in the top of upstream/custom-repositories-feature , this will help to avoid external merge problems and get easy to review the changes

File: src/components/form/WizardRepositoryTable.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          icon="repository"
+          title="No custom repositories available"
+          body="Add custom repositories to build RHEL for Edge images with additional packages."
+          primaryAction={{
+            text: 'Custom repositories',
+            href: paths['repositories'],
+          }}
+          secondaryActions={
+            [
+              //{
```
Comment (adeliaferreira) [inline src/components/form/WizardRepositoryTable.js:78]: If this is not used now, can we remove and add late when need?

**Review by adeliaferreira — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (matoval) [reply .husky/pre-commit:2]: This is added because I'm merging master into this branch.

**Review by resoluteCoder — APPROVED** [review]

**Pull request merged**